tests/run-make/rustc-crates-on-stable/rmake.rs

```diff
@@ -0,0 +1,42 @@
+//! Checks if selected rustc crates can be compiled on the stable channel (or a "simulation" of it).
+//! These crates are designed to be used by downstream users.
+
+use run_make_support::{cargo, run_in_tmpdir, rustc_path, source_root};
+
+fn main() {
+    run_in_tmpdir(|| {
```

Comment:
`run_in_tmpdir` is not needed, the test is executed in a temporary directory by default. (please also remove the import once you remove the function call :) ).

Reply:
done